barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -601,7 +601,7 @@ template <typename Curve_> class IPA {
         GroupElement shplonk_output_commitment;
         if constexpr (Curve::is_stdlib_type) {
             shplonk_output_commitment =
-                GroupElement::batch_mul(commitments, scalars, /*max_num_bits=*/0, /*with_edgecases=*/true);
+                GroupElement::batch_mul(commitments, scalars);
```

Comment:
cycle_group has edgecases handled by default and, given we'll never run IPA with biggroup I think it makes sense to just specialise this method rather than having another if-else branch for both stdlib curves

Reply:
ah, makes sense, thanks for fixing and clarifying it